src/main/java/fr/silverpricing/api/model/Chambre.java

```diff
@@ -1,8 +1,12 @@
 package fr.silverpricing.api.model;
```

Comment:
:warning: The createTutorial method does not handle exceptions properly and returns a generic internal server error response. (Severity: Major)